sam/auxtags.go

```diff
@@ -97,9 +97,13 @@ func NewAux(t Tag, value interface{}) (Aux, error) {
 		a = Aux{t[0], t[1], 'f', 0, 0, 0, 0}
 		binary.LittleEndian.PutUint32(a[3:7], math.Float32bits(v))
 	case Text:
-		a = append(Aux{t[0], t[1], 'Z'}, v...)
+		a = make(Aux, 3, len(v)+3)
```

Comment:
I'm surprised this makes so much of a difference!

Can you say it this way instead though, and below.

```
               a = make(Aux, len(v)+3)
               a[0], a[1], a[2] = t[0], t[1], 'Z'
               copy(a[3:], v)
```